front/src/styles/scss/applications/stdcm/_home.scss

```diff
@@ -294,6 +297,53 @@
         line-height: 24px;
         text-align: center;
       }
+
+      .pathFinding-status {
+        left: -8px;
+        position: absolute;
+        bottom: -95px;
+        width: 466px;
+        display: flex;
+        font-weight: 400;
+        justify-content: center;
+        line-height: 24px;
+        height: 72px;
+        border-radius: 8px;
+        color: var(--info60);
+        background-color: var(--info5);
+        margin-top: 16px;
+        padding-top: 23px;
+        box-shadow:
+          0 0 0 4px rgba(255, 255, 255, 1) inset,
+          0 0 0 5px rgb(112, 193, 229) inset,
+          0 3px 7px -3px rgba(28, 28, 217, 0.4);
+        &-success {
+          color: var(--success60);
+          background-color: var(--success5);
+          opacity: 0;
```

Comment:
I think this `opacity` can be dropped.

Reply:
It cannot: when the animation ends the state setter doesn't cause an immediate re-render to remove the element from the DOM, but we want to keep a zero opacity still.